examples/Billing/AddBillingSetup.php

```diff
@@ -198,8 +198,8 @@ public static function createBillingSetup(
                 'payments_profile_id' => new StringValue(['value' => $paymentsProfileId])
             ]));
         } else {
-            throw new Exception(
-                'No payments account ID or payments profile ID provided.' . PHP_EOL
+            throw new \RuntimeException(
```

Comment:
WDYT about UnexpectedValueException?
Should be better than RuntimeException.

Reply:
SGTM, done